Jenkinsfile

```diff
@@ -181,28 +184,84 @@ pipeline {
           }
         }
         stage('Release') {
-          options { skipDefaultCheckout() }
+          options {
+            skipDefaultCheckout()
+            timeout(time: 12, unit: 'HOURS')
+          }
+          environment {
+            HOME = "${env.WORKSPACE}"
+          }
           when {
             beforeAgent true
-            tag pattern: '@elastic/apm-rum@\\d+\\.\\d+\\.\\d+$', comparator: 'REGEXP'
+            beforeInput true
+            allOf {
+              branch 'master'
+              expression { return params.release }
```

Comment:
UI interaction, only when the parameters is set to true and the branch is the master one